.github/workflows/molecule-firewalld.yml

```
@@ -3,7 +3,7 @@ on:
   pull_request:
     paths:
       - "roles/firewalld/**"
-      - ".github/workflows/molecule.yml"
+      - "**/molecule.yml"
```

```suggestion
     - ".github/workflows/molecule.yml"
     - "**/*_base_config.yml"
```
I don't think this should run when e.g. the [`molecule.yml` file for the `install_xnat` tests](https://github.com/UCL-MIRSG/ansible-collection-infra/blob/main/tests/molecule/centos7_install_xnat/molecule.yml) is changed. The molecule file for each role is now [empty](https://github.com/UCL-MIRSG/ansible-collection-infra/blob/main/roles/firewalld/molecule/centos7/molecule.yml). The only additional files we need to run this workflow for should be the base configs in the `tests/` folder